chapters/hub/hub-api.md

```diff
@@ -0,0 +1,67 @@
+# Hub API
+
+Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute the user’s experience using `jina cli` for a seamless experience with the Hub.
```

Comment:
```suggestion
Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute users' experience using `jina cli` for a seamless experience with the Hub.
```

Reply:
Why is "These APIs constitute the user’s experience" not correct?

Reply:
`the` suggests that we are referring to a particular user we have referenced before.

Reply:
UnknownName is better than theUnkownName :stuck_out_tongue_winking_eye:

Reply:
Changed the line to `This guide covers installing and setting up the API for a seamless experience with the Hub.`, so we are not referring to the user explicitly anymore